packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -532,7 +572,8 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
       final String exportMethodDisplayName = isAppStoreUpload ? 'App Store' : exportMethod;
       status = globals.logger.startProgress('Building $exportMethodDisplayName IPA...');
       if (exportOptions == null) {
-        generatedExportPlist = _createExportPlist(exportMethod);
+        final BuildInfo buildInfo = await cachedBuildInfo;
```

Comment:
Is this needed? Isn't `buildInfo` already defined above?

https://github.com/flutter/flutter/blob/7c71a1f7e35b45bfb7f476276f41ae920b91b09e/packages/flutter_tools/lib/src/commands/build_ios.dart#L515